<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CurveEditor</class>
 <widget class="QDialog" name="CurveEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Curve Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="DistributionTypeLabel">
       <property name="text">
        <string>Distribution Type :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="DistributionType">
       <item>
        <property name="text">
         <string>Constant</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Uniform</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Curve</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Uniform Curve</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,1">
     <item>
      <widget class="QListWidget" name="ComponentList">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QDoubleSpinBox" name="ScaleMax">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1">
       <property name="topMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>12</number>
       </property>
       <item>
        <widget class="QFrame" name="Graph">
         <property name="focusPolicy">
          <enum>Qt::ClickFocus</enum>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
